cloudinit/sources/helpers/vmware/imc/guestcust_util.py

```diff
@@ -154,4 +154,12 @@ def get_tools_config(section, key, defaultVal):
     return retValue
 
 
+# Sets message to the VMX guestinfo.gc.status property to the
+# underlying VMware Virtualization Platform.
+def set_gc_status(gcMsg):
+    rpc = "info-set guestinfo.gc.status %s" % gcMsg
+    (out, err) = send_rpc(rpc)
+    return (out, err)
```

Comment:
This could be simpler:

```suggestion
    return send_rpc(rpc)
```

Reply:
Thanks, I will follow your comments.